### PR TITLE
blog: announce 23.05 release

### PR DESCRIPTION
**Pull request by RaitoBezarius**
This is a WIP for the 23.05 release blog post, feel free to suggest improvements and corrections (and even push them if possible).

We aim for a release around 10.30ish PM CEST.

### CODE REVIEW

File: blog/announcements.xml [hunk 1]
```diff
@@ -1,6 +1,139 @@
...
+      NixOS 23.05 released
+    </title>
+    <description>
+      <figure>
+        <a href="https://github.com/NixOS/nixos-artwork/blob/master/releases/23.05-raccoon/stoat.png">
+          <img class="inline" src="/logo/nixos-logo-23.05-raccoon-lores.png" alt="NixOS 23.05 Stoat logo" width="100" />
+        </a>
+      </figure>
+      <p>
+        Hey everyone, we are <a href="https://github.com/RaitoBezarius">Ryan Lahfa</a> and <a href="https://github.com/mweinelt">Martin Weinwelt</a>,
```
Comment (mweinelt) [inline blog/announcements.xml:16]:
```suggestion
        Hey everyone, we are <a href="https://github.com/RaitoBezarius">Ryan Lahfa</a> and <a href="https://github.com/mweinelt">Martin Weinelt</a>,
```

Comment (RaitoBezarius) [reply blog/announcements.xml:16]: I'm so sorry about this!

**Comment by GuillaumeDesforges** [comment]
- [ ] fix logo (currently broken link)
- [ ] fix metrics (currently same as previous release)

File: blog/announcements.xml [hunk 2]
```diff
@@ -1,6 +1,139 @@
...
+
+        Linux 5.15 is still a longterm kernel and remains available in NixOS.
+      </p>
+
+      <h3>Password hashing migration</h3>
+
+      <p>
+        Weak password hashes in NixOS 23.05 are now disabled. We consider password hashing methods weak, if
+        the libxcrypt project did not flag them <a href="https://github.com/besser82/libxcrypt/blob/v4.4.30/lib/hashes.conf#L41">strong</a>.
+        If your system is configured with weak hashes a script will emit a warning during activation. We expect
```
Comment (GuillaumeDesforges) [inline blog/announcements.xml:100]:
```suggestion
        If your system is configured with weak hashes, a script will emit a warning during activation. We expect
```

File: blog/announcements.xml [hunk 3]
```diff
@@ -1,6 +1,139 @@
...
+        Previously, NixOS shipped and defaulted on Linux 5.15, from now on, the new default version is Linux 6.1, a new longterm kernel.
+
+        Linux 5.15 is still a longterm kernel and remains available in NixOS.
+      </p>
+
+      <h3>Password hashing migration</h3>
+
+      <p>
+        Weak password hashes in NixOS 23.05 are now disabled. We consider password hashing methods weak, if
+        the libxcrypt project did not flag them <a href="https://github.com/besser82/libxcrypt/blob/v4.4.30/lib/hashes.conf#L41">strong</a>.
```
Comment (GuillaumeDesforges) [inline blog/announcements.xml:99]:
```suggestion
        Weak password hashes in NixOS 23.05 are now disabled. We consider password hashing methods weak if
        the libxcrypt project did not flag them <a href="https://github.com/besser82/libxcrypt/blob/v4.4.30/lib/hashes.conf#L41">strong</a>.
```

File: blog/announcements.xml [hunk 4]
```diff
@@ -1,6 +1,139 @@
...
+        <br />
+        The idea behind Bootspec is to enable new boot usecases in NixOS: UEFI Secure Boot, unifying bootloader installer scripts,
+        multiple initrds or systemd system extensions and A/B schemas.
+      </p>
+
+
+      <h3>Special Thanks</h3>
+      <p>
+        We want to personally thank <a href="https://github.com/riotbib">Lennart Mühlenmeier</a> and <a href="https://github.com/winterqt">Winter</a> for editorializing the release notes,
+        <a href="https://github.com/vcunat">Vladimír Čunát</a> for his tireless effort in managing jobsets, staging cycles and build infrastructure.
```
Comment (GuillaumeDesforges) [inline blog/announcements.xml:119]:
```suggestion
        <a href="https://github.com/vcunat">Vladimír Čunát</a> for his tireless effort in managing jobsets, staging cycles and build infrastructure,
```

**Comment by RaitoBezarius** [comment]
@hexa I am completely okay with dropping the Linux kernel highlight :).

File: blog/announcements.xml (hunk 3 above)
Comment (mweinelt) [reply blog/announcements.xml:99]: Should reference v4.4.33.

File: blog/announcements.xml [hunk 5]
```diff
@@ -1,6 +1,139 @@
...
+        <li><a href="/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="/download.html#download-nixos">ISO Image Downloads</a></li>
+      </ul>
+
+      <p>
+        The 23.05 release was made possible due to the efforts of <strong>1652</strong>
+        contributors, who authored <strong>30371</strong> commits since the previous release.
+        <br />
+        Our thanks go the contributors who also take care of the continued stability and
+        security of our stable distribution.
```
Comment (fricklerhandwerk) [inline blog/announcements.xml:71]:
```suggestion
        security of our stable release.
```
The wording is unusual, I suppose you mean patching the release branch? It's not clear though.

File: blog/announcements.xml [hunk 6]
```diff
@@ -1,6 +1,139 @@
...
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        while also being
+        <a href="https://repology.org/repositories/statistics/total">the distribution with the most packages</a>.
+        This release saw <strong>16346 new</strong> packages and <strong>13693 updated</strong> packages in nixpkgs.
```
Comment (fricklerhandwerk) [inline blog/announcements.xml:79]:
```suggestion
        This release saw <strong>16346 new</strong> packages and <strong>13693 updated</strong> packages in Nixpkgs.
```

File: blog/announcements.xml [hunk 7]
```diff
@@ -1,6 +1,139 @@
...
+        the libxcrypt project did not flag them <a href="https://github.com/besser82/libxcrypt/blob/v4.4.30/lib/hashes.conf#L41">strong</a>.
+        If your system is configured with weak hashes a script will emit a warning during activation. We expect
+        most users accounts to be set up with sha512crypt (hash prefixed with <i>$6$</i>) which we will continue to support.
+        Interactively configured passwords can be updated using <strong>passwd</strong>, new password hashes can be generated through <strong>mkpasswd</strong>.
+      </p>
+
+      <h3>Bootspec (RFC-125)</h3>
+
+      <p>
+        As part of standardization efforts in <a href="">RFC-125</a>, also called "Bootspec", all new users have a <i>boot.json</i> file in their system top-level derivation,
```
Comment (fricklerhandwerk) [inline blog/announcements.xml:108]:
```suggestion
        As part of standardization efforts in <a href="">RFC 125</a>, also called "Bootspec", all new users have a <code>boot.json</code> file in their system top-level derivation,
```

File names and other code tokens should be monospace.

**Review by fricklerhandwerk — COMMENTED** [review]
Congratulations to a successful new release!

File: blog/announcements.xml [hunk 8]
```diff
@@ -1,6 +1,139 @@
...
+          <img class="inline" src="/logo/nixos-logo-23.05-raccoon-lores.png" alt="NixOS 23.05 Stoat logo" width="100" />
+        </a>
+      </figure>
+      <p>
+        Hey everyone, we are <a href="https://github.com/RaitoBezarius">Ryan Lahfa</a> and <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        the release managers for this stable release and we are very proud to announce the public availability of NixOS 23.05 “Stoat”.
+      </p>
+
+      <p>
+        This release will receive bugfixes and security updates for seven months (up until 2023-12-30).
```
Comment (vcunat) [inline blog/announcements.xml:21]: Nit: why not 2023-12-31?

**Pull request merged**